speech/spec/speech_samples_spec.rb

```diff
@@ -64,12 +75,19 @@
     file = @bucket.upload_file @audio_file_path, "audio.raw"
     path = "gs://#{file.bucket}/audio.raw"
 
-    expect {
+    capture do
       speech_async_recognize_gcs project_id:   @project_id,
                                  storage_path: path
-    }.to output(
-      "Operation started\nTranscription: how old is the Brooklyn Bridge\n"
-    ).to_stdout
+    end
+
+    expect(captured_output).to include "Operation started"
+    expect(captured_output).to include "Transcription: how old is the Brooklyn Bridge"
+    expect(captured_output).to include "Word: how 0 0.3"
```

Comment:
FYI I went ahead and included the exact value.

If the start/end time of the words in this audio change... something's probably very wrong! 😄